python/test/unit/language/test_core.py

```diff
@@ -2061,7 +2061,7 @@ def kernel(X, Z, BLOCK: tl.constexpr):
         'argmax-tie-break-left': np.argmax,
     }[op]
     if 'tie-break-left' in op:
-        x[3:10] = numpy_op(x)
+        x[3:10] = x[numpy_op(x)]
```

Comment:
how is this related?

Reply:
1) The previous behavior now errors in numpy 2.0 due to out of bound failures.
2) I think the previous behavior is not what is intended in this test

reasoning:
1) numpy_op in this case is argmax/argmin. So it returns the index at which the max occurs. In the case of an input array with length > 256, the argmax can be out of bounds for an int8.
2) the test is testing "tie-break-left", so we presumably want duplicate values so we can break ties. If we imagine we are testing argmax and the input values are floats and the largest value is `1e6` at position 100, then the previous behavior sets `x[3:10] = 100`, which does not provide duplicate maximum values for the test. Instead, I expect that intended behavior was to set `x[3:10] = 1e6` so that there are multiple values with that maximal (or minimal) value.

Reply:
`np.argmax` returns the index of the maximum element in `x`, so I guess overwriting the elements with the index doesn't make a lot sense. They are supposed to be replaced by the value instead?